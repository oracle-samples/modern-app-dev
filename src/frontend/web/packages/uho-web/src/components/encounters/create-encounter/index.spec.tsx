/*
 * Copyright (c) 2023 Oracle and/or its affiliates.
 * Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracl.com/licenses/upl/
 */
import { CreateEncounter } from '.';
import { h } from 'preact';
import { queryClient, snapshot } from '../../../../test/jetHelper';
import { QueryClientProvider } from '@tanstack/react-query';

snapshot(
  'CreateEncounter',
  <body>
    <QueryClientProvider client={queryClient}>
      <CreateEncounter appointmentId={1} onClose={console.log} patientId={1} providerId={1} show={true} />
    </QueryClientProvider>
  </body>
);
