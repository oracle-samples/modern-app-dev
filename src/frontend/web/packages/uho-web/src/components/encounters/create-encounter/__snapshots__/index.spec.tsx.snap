// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot tests for CreateEncounter Snapshot: Component - CreateEncounter 1`] = `
<DocumentFragment>
  <div
    data-oj-binding-provider="preact"
  >
    <div>
      <oj-surrogate
        class="oj-component-initnode oj-complete"
        data-oj-binding-provider="none"
      />
    </div>
  </div>
  <div
    id="__oj_layerhost_container"
  />
</DocumentFragment>
`;
